src/ProductConstructionService/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Starting the service locally
+
+To run the Product Construction Service locally, set the `ProductConstructionService.AppHost` as Startup Project, and run with F5
+
+# Instructions for recreating the Product Construction Service
```

Comment:
I am wondering what's the point of having these commands in a markdown rather than just have `Create-Azure-Resources.ps1`?

Reply:
Added a script `provision.ps1` and updated the instructions